calm/draft/2024-08/meta/core.json

```diff
@@ -53,6 +53,17 @@
             "$ref": "interface.json#/defs/interface-type"
           }
         },
+        "domains": {
+          "type": "object",
+          "patternProperties": {
+            "^[a-zA-Z0-9_]+$": {
```

Comment:
Nice! I didn't know you could do this :octocat:

Reply:
`JSONSchema` is a hard language to learn 😄